user_docs/en/changes.t2t

```diff
@@ -14,8 +14,8 @@ There is a new action to open a reviews webpage for the selected add-on.
 The audio output device and ducking mode options have been removed from the "Select Synthesizer" dialog.
 They can be found in the audio settings panel which can be opened with ``NVDA+control+u``.
 
-eSpeak-NG, LibLouis braille translator, and Unicode CLDR have been updated.
-New Thai and Romanian braille tables are available.
+eSpeak-NG, LibLouis Braille translator, and Unicode CLDR have been updated.

```

Comment:
please decapitalise braille, it's more standard. LibLouis docs doesn't capitalise it either